builder/hyperv/iso/builder.go

```diff
@@ -265,6 +265,9 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 			SwitchVlanId: b.config.SwitchVlanId,
 		},
 
+		&hypervcommon.StepSetBootOrder{
```

Comment:
is this a step that would work only for `iso`?

Reply:
it also works for vmcx and the latest revision of this MR has it too